src/common/get-user-act-no.js

```diff
@@ -0,0 +1,7 @@
+import getAllUsers from "./get-all-users";
+export const getUserByAccountNumber = (accountNumber) => {
+  const usersArr = getAllUsers();
+  const user = usersArr.find((user) => user.accountNumber === accountNumber);
+  return user;
+};
+export default getUserByAccountNumber;
```

Comment:
You already did a named export on line 2. Either remove the export keyword on line 2, or remove this line to use the named export.

Reply:
Merge this file and `get-all-users.js` into one file. They both contain the same logical information, so it makes sense to put them in the same file.
You should use named export for the functions, i.e. remove `export default ...`